extensions/wikia/DesignSystem/DesignSystemHelper.class.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+use \Wikia\Logger\WikiaLogger;
+
+class DesignSystemHelper {
+
+	const DESIGN_SYSTEM_DIR = __DIR__ . '/bower_components/design-system';
+	const ASSETS_DIR = self::DESIGN_SYSTEM_DIR . '/assets';
+
+	private static $svgCache = [];
+
+	/**
+	 * @desc Returns SVG content
+	 *
+	 * @param string $group
+	 * @param string $name
+	 * @param string $classNames
+	 * @param string $alt
+	 *
+	 * @return string
+	 */
+	public static function getSvg( $group, $name, $classNames = '', $alt = '' ) {
+		$xml = self::getCachedSvg( $group, $name );
+
+		if ( $xml instanceof SimpleXMLElement ) {
+			/* @var $xml SimpleXMLElement */
+
+			if ( !empty( $classNames ) ) {
+				$xml->addAttribute( 'class', $classNames );
+			}
+
+			if ( !empty( $alt ) ) {
+				$xml->addAttribute( 'alt', $alt );
+			}
+
+			return $xml->asXML();
+		} else {
+			WikiaLogger::instance()->error( 'Design System SVG could not be loaded', [
+				'group' => $group,
+				'name' => $name
+			] );
+
+			$fallback = '';
+
+			if ( !empty( $alt ) ) {
+				$fallback = $alt;
+			}
+
+			if ( !empty( $name ) ) {
```

Comment:
Your previous version suggested you wanted to use `$alt` if it was available, and `$name` if it wasn't, but if both are set `$name` will be used as this will overwrite it. Is that the intent, or should this be an `elseif`?